src/js/chartAuthoring.js

```diff
@@ -102,7 +116,8 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                         type: "fluid.transforms.free",
                                         args: ["{that}.model.dataSet"],
                                         func: "floe.chartAuthoring.dataEntriesToPieChartData"
-                                    }
+                                    },
+                                    backward: "never"
```

Comment:
This line is un-necessary since "fluid.transforms.free" doesn't support the backwards transformation.